File: src/main/java/com/lessmarkup/userinterface/model/user/ResetPasswordEmailModel.java
package com.lessmarkup.userinterface.model.user;

import com.lessmarkup.interfaces.system.MailTemplateModel;

public class ResetPasswordEmailModel extends MailTemplateModel {
    private String resetUrl;
    private String siteName;

    public String getResetUrl() {
        return resetUrl;
    }

    public void setResetUrl(String resetUrl) {
        this.resetUrl = resetUrl;
    }

    public String getSiteName() {
        return siteName;
    }

    public void setSiteName(String siteName) {
        this.siteName = siteName;
    }
}

File: src/main/scala/com/lessmarkup/userinterface/model/user/ResetPasswordEmailModel.scala
/*
 * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
 * If a copy of the MPL was not distributed with this file, You can obtain one at
 * http://mozilla.org/MPL/2.0/.
 */

package com.lessmarkup.userinterface.model.user

import com.lessmarkup.interfaces.system.MailTemplateModel

class ResetPasswordEmailModel extends MailTemplateModel {
  private var resetUrl: String = null
  private var siteName: String = null

  def getResetUrl: String = resetUrl

  def setResetUrl(resetUrl: String) {
    this.resetUrl = resetUrl
  }

  def getSiteName: String = siteName

  def setSiteName(siteName: String) {
    this.siteName = siteName
  }
}